src/coreclr/vm/appdomain.hpp

```diff
@@ -1103,7 +1103,7 @@ class BaseDomain
 
 #endif // DACCESS_COMPILE && !CROSSGEN_COMPILE
 
-    CLRPrivBinderCoreCLR *GetTPABinderContext() {LIMITED_METHOD_CONTRACT;  return m_pTPABinderContext; }
+    DefaultAssemblyBinder *GetTPABinderContext() {LIMITED_METHOD_CONTRACT;  return m_pTPABinderContext; }
```

Comment:
I would like if we could rename this method to `GetDefaultBinderContext` - or even better `GetDefaultBinder`. And in general get rid of the "binder context" term, and just use "binder".

The "TPA" part should be more or less an implementation detail. I know we're now keeping it on per-assembly level, but eventually that should go away as well (as a term and maybe even as the functionality - I'm having certain doubts about its usefulness - but that's a different discussion).

Reply:
Yes. It might be ok to use "TPA" when referring to the actual set of assemblies, but there is really no such thing a s TPA binder - there is DefaultBinder who's binding behavior mostly guided by TPA set, but that is an implementation detail for the binder.